code/script.js

```diff
@@ -26,22 +34,122 @@ const showMessage = (message, sender) => {
       </section>
     `
   }
-  // This little thing makes the chat scroll to the last message when there are too many to be shown in the chat box
   chat.scrollTop = chat.scrollHeight
 }
 
+const nextQuestion = (message) => {
+  console.log('questionNumber', questionNumber)
+
+  if (questionNumber === 1) {
+    userReply(message)
+    input.value = ''
+    setTimeout(() => showDrinkTypes(message), 1000)
+  } else if (questionNumber === 2) {
+    userReply(message)
+    setTimeout(() => showMenu(message), 1000)
+  } else if (questionNumber === 3) {
+    userReply(message)
+    setTimeout(() => showPrice(message), 1000)
+  } else {
+    userReply(message)
+    setTimeout(thankYou, 1000)
+  }
+}
+
 // Starts here
 const greeting = () => {
-  showMessage(`Hello there, What's your name?`, 'bot')
-  // Just to check it out, change 'bot' to 'user' here 👆
+  questionNumber = 1
+  botReply(`Hey, my name is Ben. I'll help you placing your order. What is your name?`)
+}
+
+const showDrinkTypes = (msg) => {
+  questionNumber++
+  botReply(
+    `Great, so let's start! What kind of beer are you looking for, ${msg}?`
+  )
+
+  inputWrapper.innerHTML = `
+    <button id="lagerBtn">Lager</button>
+    <button id="palealeBtn">Pale Ale</button>
+    <button id="ipaBtn">IPA</button>
+  `
+
+  document
+    .getElementById('lagerBtn')
+    .addEventListener('click', () => nextQuestion('Lager'))
+  document
+    .getElementById('palealeBtn')
+    .addEventListener('click', () => nextQuestion('Pale Ale'))
+  document
+    .getElementById('ipaBtn')
+    .addEventListener('click', () => nextQuestion('IPA'))
+}
+
+const showMenu = (type) => {
+  questionNumber++
+
+  botReply(
+    `Perfect choice! ${type} is always good.`
+  )
+
+  if (type === 'Lager' || 'Pale Ale' || 'IPA') {
+    inputWrapper.innerHTML = `
+      <select id="select">
+        <option value="" selected disabled>➡️ Select what crate size you'd like</option>
+        <option value="s">6 bottles</option>
+        <option value="m">12 bottles</option>
+        <option value="l">24 bottles</option>
```

Comment:
Maybe the value should be the same as the content -> "12 bottles" etc, since just "m" doesn't mean much to the user :)

Reply:
Maybe add some styling to this drop down as well - to better match the rest of your app